product/reports/670_Performance by Asset Type - Middleware Servers/160_Web Metrics - hourly over last day.yaml

```diff
@@ -0,0 +1,59 @@
+---
+where_clause:
+dims:
+reserved:
+title: "EAP Web Metrics - hourly over the last day"
+conditions:
+order: Descending
+graph:
+menu_name: "EAP Web Metrics - hourly over the last day"
+rpt_group: Custom
+priority:
+col_order:
+- id
+- name
+- start_time
+- mw_aggregated_active_web_sessions_max
+- mw_aggregated_max_active_web_sessions_max
```

Comment:
Looking at your screenshot and googling a little, `max_active_web_sessions` probably is not a metric, but a configuration attribute. In your screenshot it's always `-1`, probably meaning that the number of active sessions is not limited. May be @jmazzitelli can answer better.

If it's a config value, it shouldn't be in the report nor in the agent config.

Reply:
Looking at the hawkular agent configurations, that is an actual metric. Check the config files for both [EAP6](https://github.com/hawkular/hawkular-agent/blob/master/eap6-support/hawkular-javaagent-wildfly-feature-pack-eap6/src/main/resources/featurepack/content/standalone/configuration/hawkular-javaagent-config.yaml#L93) and [EAP7](https://github.com/hawkular/hawkular-agent/blob/master/hawkular-javaagent-wildfly-feature-pack/src/main/resources/featurepack/content/standalone/configuration/hawkular-javaagent-config.yaml#L93).

Also, looking into the aggregated values within the Wildfly client, you can see as well that this is a read-only attribute as the other metrics in the subsystem. ![reports-metric](https://user-images.githubusercontent.com/613814/32095991-55c2b6f0-bb05-11e7-9ae0-51cd67bbbe61.png).

As you said, that `-1` is because the max active session setup was set to unlimited. Once the config is changed, the report prints out same value as config.
![reports-metric-2](https://user-images.githubusercontent.com/613814/32100186-718560dc-bb13-11e7-8321-8b00a1fbdff6.png)

In my opinion, I would leave this metric since it could be useful for a admin to see max-active-sessions as a threshold for the active sessions. Also, it could be useful to explain an increase/decrease of active sessions due to a change of the configuration attribute.

Reply:
Ok! If it's going to stay, the label should reflect that it's a config value. Otherwise, user will get confused with two consecutive columns whose label is very, very similar.

Reply:
@israel-hdez just saw more info from John Doyle about the required columns for that report. here the ones that must be in:
* Active Web Sessions
* Expired Web Sessions
* Rejected Web Sessions

I think it is more clear now.
Regarding to the hourly report, it makes sense to add min/max /avg columns. In the minute-ranged one, it doesn't. As you said before, previous metrics are collected every minute thus metric values would be exactly the same in avg, min and max.